README.md

```diff
@@ -261,6 +261,11 @@ They will add and remove packages and could leave your OS in a bad
 state.  You should not use them on your personal computer.  Use in a vm
 or container.
 
+Localization
+--------
+
+All translations are generated using [Weblate](https://hosted.weblate.org/projects/tuxemon/).
+For details check the README in `mods/tuxemon/l18n`
```

Comment:
Do we need this section?

Reply:
Nope, I'm removing it. That's too much redundant info. We have the README in `/mods/tuxemon/l18n` and the section in CONTRIBUTING.